stix_shifter/stix_translation/src/modules/splunk/stix_to_splunk.py

```diff
@@ -30,19 +31,24 @@ def transform_query(self, data, options, mapping=None):
         query_object = generate_query(data)
         data_mapper = options.get('data_mapper')
         mapping = options.get('mapping')
+        fields = options.get('fields')
 
         if not data_mapper:
             data_mapper = 'cim'
 
-        data_mapper_module_name = ''.join(["stix_shifter.stix_translation.src.modules.", data_mapper, ".", data_mapper, "_data_mapping"])
-
-        try:
-            data_mapper_module = importlib.import_module(data_mapper_module_name)
-            data_model_mapper = data_mapper_module.mapper_class(mapping)
-        except ModuleNotFoundError:
-            raise NotImplementedError(f"Module {data_mapper_module_name} not implemented")
-        except AttributeError:
-            raise NotImplementedError(f"Module {data_mapper_module_name} does not implement mapper_class attribute")
+        if data_mapper != 'cim':
```

Comment:
we can do something like: 
```
if not data_mapper:
    data_mapper = 'cim'
    data_mapper_module = cim_data_mapping
    data_model_mapper = data_mapper_module.mapper_class(mapping,fields)
else:
  # put the code for if data_mapper != 'cim':
```
this way we can avoid one extra conditional checking.

Reply:
let me know if that works